dune-project

```diff
@@ -38,5 +38,4 @@ Changes:
     conf-perl
     cppo
   )
-  (conflicts cil)
```

Comment:
Are we sure this conflict is also gone w.r.t. e.g. the `cilly` library?

Reply:
Good point, I forgot about that. I'll add the conflict back. Other than the cilly executable there shouldn't be any issue.